sql/sql_rewrite.cc

```diff
@@ -385,7 +385,7 @@ void mysql_rewrite_create_alter_user(THD *thd, String *rlb,
 
   while ((tmp_user_name= user_list++))
   {
-    if ((user_name= get_current_user(thd, tmp_user_name)))
+    if ((user_name= get_current_user(thd, tmp_user_name, true)))
```

Comment:
Looks good, just one more thing I noticed now: these changes should be in ifdef wsrep blocks.

Reply:
Yup, exactly the same thing I noticed in the meantime :) Fixed.